pkg/component/etcd/service/values_helper.go

```diff
@@ -33,6 +33,7 @@ func GenerateValues(etcd *druidv1alpha1.Etcd) Values {
 		BackupPort:        pointer.Int32Deref(etcd.Spec.Backup.Port, defaultBackupPort),
 		ClientPort:        pointer.Int32Deref(etcd.Spec.Etcd.ClientPort, defaultClientPort),
 		ClientServiceName: utils.GetClientServiceName(etcd),
+		ClientAnnotations: etcd.Spec.Etcd.ClientService.Annotations,
```

Comment:
service name is captured as `ClientServiceName`, shouldn't annotations be named as `ClientServiceAnnotations`?

Reply:
Yes, that makes things more clear
Updated